### DISCUSSION TITLE
Does this require MKVToolNix to be installed?

### DISCUSSION THREAD

**Discussion by Squirreljester** [jlesage/docker-mkvcleaver#7] [Q&A]
I just discovered MKVToolNix docker which brought me to this project. I'd love to try it, but I had a couple questions. I'd be deploying this to my Unraid server. Right now I have MKVToolNix installed on my Windows client machine.

Does this require MKVToolNix to be installed on my Windows client machine?
Can I use the MKVToolNix docker container instead of the instance installed on my Windows client machine?
If I deploy both containers, is there a way I can access the command line functions of either this or the MKVToolNix docker for scripting things this project can't do?

Thanks in advance for any assistance.

**Comment by jlesage** [comment]
> Does this require MKVToolNix to be installed on my Windows client machine?

Containers are self-contained, thus you don't need any dependencies on your Windows machine (except Docker itself).

> Can I use the MKVToolNix docker container instead of the instance installed on my Windows client machine?

Yes you can.

> If I deploy both containers, is there a way I can access the command line functions of either this or the MKVToolNix docker for scripting things this project can't do?

Yes, you can access MKVToolNix cli commands.  They are installed inside the container.
